interface/usergroup/user_info.php

```diff
@@ -83,5 +122,103 @@ function(data)
 </HTML>
 <?php
   //  da39a3ee5e6b4b0d3255bfef95601890afd80709 == blank
+
+if (isset($_FILES)) {
+  //images will be saved with their uid
+  $uid = $_SESSION['authId'];
+    //MAKE THE UPLOAD DIRECTORY IF IT DOESN'T EXIST
+  if (realpath("../../profile_pictures/")) {
+      
+  }
+  else {
+    mkdir("../../profile_pictures/", 0755);
+  }
+  //for profile picture upload
+  //mime check done.
+  //size check done.
+  //extension check done.
+  //if any validation needed be added, please add it below.
+  $bool = 0;
+  $target_file =  basename($_FILES["profile_picture"]["name"]);
+  $imageFileType = strtolower(pathinfo($target_file,PATHINFO_EXTENSION));
+  $verify_image = getimagesize($_FILES["profile_picture"]["tmp_name"]);
+  if($verify_image) {
+    $mime = $verify_image["mime"];
+    $mime_types = array('image/png',
+                            'image/jpeg',
```

Comment:
Just need to know if it is an image...your code works, but getimagesize returns false for non images, so....
```
if(isset($_POST["submit"])) {
    $check = getimagesize($_FILES["fileToUpload"]["tmp_name"]);
    if($check !== false) {
        echo "File is an image - " . $check["mime"] . ".";
        $uploadOk = 1;
    } else {
        echo "File is not an image.";
        $uploadOk = 0;
    }
```

Reply:
already i have made the validation, if it is not a image then $bool = 0;

Reply:
if $bool=0 then no file upload takes place, and no db records get updated.

Reply:
Gotcha